cmd/blockchaininfo/nats_subscriber.go

```diff
@@ -107,15 +107,29 @@ func receiveContractUpdates(msg *nats.Msg, contractMsg []byte, scheme proto.Sche
 			contractMsg = nil
 		}
 	}
-
 	return contractMsg
 }
 
+func ConcatenateContractTopics(contractAddress string) string {
+	return blockchaininfo.ContractUpdates + contractAddress
+}
+
+//nolint:unused // because this function will be called in some other place.
```

Comment:
Please, on't use `nolint` for ignoring the linter's error. The issue can be fixed by using this function in test.